frontend/src/constants/propertySteps.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Land steps enum, provides a list of steps corresponding to submittig bare land.
+ */
+export enum LandSteps {
```

Comment:
Are you going to have another enum for when associated land will only be 2 steps?

Reply:
Just so Devin doesn't get the blame for any of my changes these are from my PR's